### PR TITLE
Player Housing Prototype

### PR DESCRIPTION
**Pull request by GarOfMeridian**
* Buy a house buy saying "buy house" in Familiars. Costs 500k.

* Get given a key that, when used in Familiars, takes you to your house.

* Buy extra rooms by saying "buy room" inside. Costs 1 million for the
first expansion. The second expansion adds a new room on a second floor,
and costs 2 million. The final expansion adds a second room on the
second floor for 3 million.

* Rooms 3 and 4 are on the second floor of your house - go up the stairs
to reach them. This is the first application of a new modular technique
that actually gives us the illusion of a functioning Z axis. Your second
floor is directly above your first floor - it just changes heights and
loads whichever items are on that floor as you use the stairs.

This is an extremely basic prototype for testing that this actually works on a
live server.

Total plan for housing:

- Any yells in Tos will be heard in houses

- Shared neighborhood instance where you can see neighboring houses

- Large amount of customization options (objects, textures)

- Player housing should become a central location for all of a person's
characters

- Vault accesses for a hefty price (for all characters that use the house,
i.e. sharing vaults)

Basically, it's going to be something players can work on building up
over long periods of time to make very cool and functional.

### CODE REVIEW

**Comment by treymd** [comment]
housekey.kod(22): error: parse error on token if you use it in Familiars."
housekey.kod(28): error: Unknown identifier HouseKey_desc_rsc
